ユーザー機能　ゲストログインの導入 #46 #48

fujii-kosuke-36 opened the pull request:
gem sorceryをインストールして認証機能を導入しました。
user_sessions　controllerを作成してゲストログインを導入しました。

fujii-kosuke-36 committed:
gem sorceryをインストールして認証機能を導入しました. #46 #48

sorceryを導入　ユーザーテーブル作成　ユーザーモデルのバリデーション

fujii-kosuke-36 committed:
gem sorceryをインストールして認証機能を導入しました. #46 #48

sorceryを導入　ユーザーテーブル作成　ユーザーモデルのバリデーション

fujii-kosuke-36 committed:
ゲストログインの導入 #46

user_sessions controllerを作成してゲストログイン機能を導入しました。ルーティングも追加。

fujii-kosuke-36 committed:
ゲストログインの導入 #46

user_sessions controllerを作成してゲストログイン機能を導入しました。ルーティングも追加。

fujii-kosuke-36 committed:
Merge branch 'develop' into feature/create-user_#46_#48

itomiki35 commented:
・Usersテーブルにguestカラムを追加してほしいです
・ヘッダーのゲストログインにリンクを作って仮のページに飛ばすようにしてほしいです
（コンソールで試すでも良いのでお任せします。）

fujii-kosuke-36 committed:
users_tableにguestカラムを追加 #46

users_tableにgesutカラムをboolean型で追加

fujii-kosuke-36 committed:
コンフリクト解消　ゲストログイン機能の修正 #46

コンフリクトを解消してゲストログイン機能が作動することを確認

fujii-kosuke-36 committed:
コンフリクト解消 #46

コンフリクト解消
Merge remote-tracking branch 'refs/remotes/origin/feature/create-user_#46_#48' into feature/create-user_#46_#48

ayakuma309 commented:
LGTMです
